Resources/views/CRUD/edit_orm_many_to_many.html.twig

```diff
@@ -9,7 +9,8 @@ file that was distributed with this source code.
 
 #}
 {% if sonata_admin.field_description.hasassociationadmin %}
-    <div id="field_container_{{ id }}" class="field-container {{ form.vars["attr"]["class"] is defined ? form.vars["attr"]["class"] }}">
+    <div id="field_container_{{ id }}"
+		class="field-container {{ form.vars["attr"]["class"] is defined ? form.vars["attr"]["class"] }}">
```

Comment:
There is even a comic for that one :

![for the lulz](http://evadeflow.com/wp-content/uploads/2011/03/TabsSpacesBoth.png)